### PR TITLE
Database

### PR DESCRIPTION
**Pull request by azNodirbek**
check database and file connections

### CODE REVIEW

File: contact-us.php [hunk 1]
```diff
@@ -1,7 +1,10 @@
+
```
Comment (arakhimiy) [inline contact-us.php:1]: you should add new line

File: customer/dbconnect.php [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<?php
```
Comment (arakhimiy) [inline customer/dbconnect.php:2]: you should add new line

File: links.php [hunk 3]
```diff
@@ -17,4 +17,6 @@
 <link href="css/main.css" rel="stylesheet" >
```
Comment (arakhimiy) [inline links.php:17]: Something wrong ,please check it!

File: contact-us.php (hunk 1 above)
Comment (azNodirbek) [reply contact-us.php:1]: done

File: customer/dbconnect.php (hunk 2 above)
Comment (azNodirbek) [reply customer/dbconnect.php:2]: Done

File: links.php (hunk 3 above)
Comment (azNodirbek) [reply links.php:17]: I have checked, it is cool

**Review by khomitova — APPROVED** [review]

**Pull request merged**